docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/metatag/Group/CgdpAdvanced.php

```diff
@@ -15,5 +15,5 @@
  * )
  */
 class CgdpAdvanced extends GroupBase {
-  // Nothing here yet. Just a placeholder class for a plugin.
```

Comment:
Technically this comment should have been changed in the commit where it was added.